### DISCUSSION TITLE
Why do a lot of csprojs still target netstandard2.1?

### DISCUSSION THREAD

**Discussion by MagicAndre1981** [dotnet/maui#8744] [Q&A]
[I followed the guide](https://github.com/dotnet/maui/blob/main/.github/DEVELOPMENT.md#cake=) to locally build MAUI and in VS2022 I see no a lot of errors for configuration nestandard2.1:

![image](https://user-images.githubusercontent.com/8274816/179080468-809f4203-9452-4863-9490-af4d76c911e3.png)

Why do a lot of csprojs still target netstandard2.1?

**Comment by Xyncgas** [comment]
Because it's the latest standard

**Reply by MagicAndre1981** [reply]
MAUI runs at least on .net6, so only net6.0 should be targeted. [net standard 2.1](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/) is deprecated in favor for newer .net versions:

> While .NET Standard 2.1 will be the last version of .NET Standard, .NET 5 and all future versions will continue to support .NET Standard 2.1 and earlier. You should think of net5.0 (and future versions) as the foundation for sharing code moving forward.
